sentry-core/src/performance.rs

```diff
@@ -1143,8 +1159,12 @@ impl Iterator for TraceHeadersIter {
 
 /// A container for distributed tracing metadata that can be extracted from e.g. the `sentry-trace`
 /// HTTP header.
-#[derive(Debug, PartialEq)]
-pub struct SentryTrace(protocol::TraceId, protocol::SpanId, Option<bool>);
+#[derive(Debug, PartialEq, Clone, Copy, Default)]
+pub struct SentryTrace {
+    pub(crate) trace_id: protocol::TraceId,
+    pub(crate) span_id: protocol::SpanId,
+    pub(crate) sampled: Option<bool>,
```

Comment:
Looks like `sampled` is not being used outside this module, so you should be able to make it private I think.

```suggestion
    pub(crate) trace_id: protocol::TraceId,
    pub(crate) span_id: protocol::SpanId,
    sampled: Option<bool>,
```

Also [nit] I would consider just adding accessors for the `trace_id` and `span_id` since it doesn't look like you are mutating them from outside (but I suppose that is just my personal preference that I like to keep struct fields private and access through methods; since they are already only `pub(crate)` it doesn't matter that much so feel free to ignore this suggestion)

Reply:
Also I think this change to named fields makes the code way more readable 🎉